### PR TITLE
chore(deps): Configure renovate commit titles

### PR DESCRIPTION
**Pull request by ChristophWurst**
* `Bump` instead of `update dependency`
* Include current version

Ref https://github.com/nextcloud/groupware/issues/63#issuecomment-1443055107

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
 		"dependencies",
 		"3. to review"
 	],
+	"commitMessageAction": "Bump",
+	"commitMessageTopic": "{{depName}}",
+	"commitMessageExtra": "from {{currentVersion}} to {{#if isPinDigest}}{{{newDigestShort}}}{{else}}{{#if isMajor}}{{prettyNewMajor}}{{else}}{{#if isSingleVersion}}{{prettyNewVersion}}{{else}}{{#if newValue}}{{{newValue}}}{{else}}{{{newDigestShort}}}{{/if}}{{/if}}{{/if}}{{/if}}",
```
Comment (st3iny) [inline renovate.json:19]: I could imagine `currentVersion` to be undefined in some cases (pinned digests). We'll see how it behaves in practice. The official documentation does not elaborate further on the value of this variable.

**Review by st3iny — APPROVED** [review]
LGTM!

File: renovate.json (hunk 1 above)
Comment (ChristophWurst) [reply renovate.json:19]: I took inspiration from https://github.com/DefectDojo/django-DefectDojo/blob/master/.github/renovate.json

Comment (ChristophWurst) [reply renovate.json:19]: Bildschirmfoto vom 2023-02-24 10-38-53

seems to work ok

**Pull request merged**